src/sentry/workflow_engine/migration_helpers/issue_alert_conditions.py

```diff
@@ -34,8 +35,16 @@
 ](enable_reverse_lookup=False)
 
 
-def translate_to_data_condition(data: dict[str, Any], dcg: DataConditionGroup):
-    translator = data_condition_translator_registry.get(data["id"])
+def translate_to_data_condition(
```

Comment:
not sure if an ANY DCG with `EveryEventCondition` should just have the rest of the conditions removed, currently it will always just be firing on every event while ignoring the other conditions in that DCG lol

it would certainly be less noisy if we just dropped `EveryEventCondition`